Makefile

```diff
@@ -69,7 +69,7 @@ export LOGCHECK # so hack scripts can use it
 
 CODE_GENERATOR_VER := v2.0.0-alpha.1
 CODE_GENERATOR_BIN := code-generator
-CODE_GENERATOR := $(TOOLS_GOBIN_DIR)/$(CODE_GENERATOR_BIN)-$(CODE_GENERATOR_VER)
+CODE_GENERATOR := $(TOOLS_GOBIN_DIR)/$(CODE_GENERATOR_BIN)-$(CODE_GENERATOR_VER) 
```

Comment:
seems like your editor pushed some spaces. can you please remove?

Reply:
Done